scripts/irods/test/test_irsync.py

```diff
@@ -747,7 +749,11 @@ def test_irsync_does_not_create_collections_on_dry_run_issue_7774(self):
         self.user0.assert_icommand(['itouch', f'{self.user0.session_collection}/{source_coll}/three'])
 
         # Perform the irsync
-        self.admin.assert_icommand(['irsync', '-rl', f'i:{self.user0.session_collection}/{source_coll}', f'i:{self.user0.session_collection}/{dest_coll}' ], 'STDOUT', ['one', 'two', 'three'])
+        self.user0.assert_icommand(['irsync', '-rl', f'i:{self.user0.session_collection}/{source_coll}', f'i:{self.user0.session_collection}/{dest_coll}' ], 'STDOUT', ['one', 'two', 'three'])
```

Comment:
Consider constructing the paths one time and reusing them. It shortens the line length and makes it easier to notice the same paths are being used in multiple assertions.